### DISCUSSION TITLE
[FR] Media Player - Volume limits

### DISCUSSION THREAD

**Discussion by Shiu** [Clooos/Bubble-Card#919] [Feature requests]
The Media Player card needs a volume limit. 

Media Player volume currently goes from 0 to 100% volume on the slider. If I were to accidentally set the volume to 100% I would blow out my speakers, so I've removed the Media Player card for now. I consider it a necessity to be able to set volume limits for the slider, and then possibly be able to bypass the limit by using separate + and - buttons, but the bypass is not a requirement.

I rarely consider something a necessity, but the current implementation is simply too risky to use.

**Comment by exetico** [comment]
Duplicate of https://github.com/Clooos/Bubble-Card/discussions/304

**Reply by MrBearPresident** [reply]
Not completly, this is on the media player. 
#304 is on the button, slider card.

**Reply by exetico** [reply]
But you're stating "on the slider", but I get it. I'd just like to mention it :)
